Sources/Plasma/PubUtilLib/plPipeline/plTextFont.cpp

```diff
@@ -57,6 +57,11 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 
 #include "plDebugText.h"
 
+#if defined(HS_BUILD_FOR_MACOS)
```

Comment:
Hmm... I thought this define was changed to `HS_BUILD_FOR_APPLE`, or am I misremembering things?

Reply:
`HS_BUILD_FOR_APPLE` for stuff that's Apple-y, `HS_BUILD_FOR_MACOS` for stuff that is strictly macOS.

CoreGraphics, I _think_ would fall under Apple, since it's also available on iOS?

Reply:
Yeah, you're right. If we ever did an iOS version (which I'd love to at least hack on someday), this code would work. I'll update the check.

Reply:
This macro is still checking `HS_BUILD_FOR_MACOS` instead of `HS_BUILD_FOR_APPLE`